### PR TITLE
feat: add tool cleanup pr comments

### PR DESCRIPTION
**Pull request by sqin2019**

### CODE REVIEW

File: .github/workflows/tool_cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                req = fs.readFileSync(
+                  `tool.yaml`,
+                  { encoding: "utf8" }
+                );
+
+              body = `**\`Access on Demand\`** - 🟩 **\`Tool\`** request succeeded.
+
+            <details>
+            <summary>Details</summary>
+            Executed "cleanup" commands in the request below, or skipped if "cleanup" commands not found.
```
Comment (sqin2019) [inline .github/workflows/tool_cleanup.yml:114]: A little awkward when no commands were found...

Screenshot 2023-07-20 at 3 32 50 PM
Screenshot 2023-07-20 at 3 32 41 PM

File: .github/workflows/tool_cleanup.yml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Reusable workflow that handles tool request cleanup.
+name: 'aod-tool-cleanup'
+
+# Support below trigger:
+# pull_request:
+#   types: 'closed'
+#   paths: 'tool.yaml'
```
Comment (yolocs) [inline .github/workflows/tool_cleanup.yml:21]: Is this intentional? Or should live somewhere else?

File: .github/workflows/tool_cleanup.yml (hunk 1 above)
Comment (yolocs) [reply .github/workflows/tool_cleanup.yml:116]: https://github.com/abcxyz/access-on-demand/issues/79

**Review by yolocs — APPROVED** [review]

File: .github/workflows/tool_cleanup.yml (hunk 2 above)
Comment (sqin2019) [reply .github/workflows/tool_cleanup.yml:21]: Removing.

**Review by yolocs — APPROVED** [review]

**Pull request merged**